Assets/HoloToolkit/Input/Scripts/Voice/SpeechInputSource.cs

```diff
@@ -71,7 +71,7 @@ protected override void Start()
             }
         }
 
-        void Update()
+        protected virtual void Update()
```

Comment:
Any Unity methods should be either private or protected if inheriting from base class.